cpu/cpupower_monitor.py

```diff
@@ -28,7 +28,7 @@
 class CpupowerMonitor(Test):
 
     """
-    Test to validate idle states using cpupower monitor tool.
+    Test to validate idle states using cpupowe monitor tool.
```

Comment:
can you please remove this change from this commit as earlier commit address this

Reply:
Oops, did not realise both commits had this change. Thanks for noticing.
Have removed this change from commit ea3efc1 .